doc/Type/IO/Path.pod6

```diff
@@ -976,8 +976,6 @@ Defined as:
 Coerces L«C<.basename>|/routine/basename» to L<Int|/type/Int>. L<Fails|/routine/fail>
 with C<X::Str::Numeric> if base name is not numerical.
 
-X<|symlink (sub)>
-X<|symlink (method)>
 =head2 routine symlink
```

Comment:
These have simply disappeared?

Reply:
I believe they are bogus. The symlink routine is already indexed as subroutine because of the `=head2 routine symlink` syntax, there is no need to double/triple index it. Others don't have it, just look at the https://github.com/Raku/doc/blob/bab14801aa96c36e357645bf9212f74a3acab59b/doc/Type/IO/Path.pod6 <- or others, no other method/subroutine is indexed this way.

So I believe it's just the author wanting to make an anchor but being ignorant (and those who reviewed it) to how the anchors are indexed, resulting in those two - just another consequence of indexing being obscure thing for years.

Reply:
The issue is that you're mixing several different issues in a single PR. One thing is adding categories to terms that didn't have one. That's one thing. Another is fixing issues with indexing. That's another thing. Changing categories is yet another. And finally, dropping indexing for whatever you think does not need to be indexed is yet another. Every one of them should have been given some time and, if possible, feedback from the community. In general, I'm against deleting things from the documentation (and have said it in so many words in many PRs). In particular, indexing terms that can be either subs or methods *in the same place* under two different terms, is possibly not awesome, so I might agree with you. However, that's a different discussion, would need a separate issue. Since this PR has 90 different files and lots of changes, it's simply impossible to give it the attention it deserves (even more so if you merge it during someone's holidays)

Reply:
I agree things always could've been done better, but I am not sure what is your point.

Yes, you had a vacation and the PR was merged, but this work was here for over a year and nobody paid attention or wanted to instigate a discussion to revive it, nothing. So, in the end, you either get something working and then polish it until it's good or you wait forever waiting for some miracle perfect solution.

Is there something particular you ask from me?

Reply:
What I'm not sure is that even if I make the point it's going to make any difference. So, again, at this point what I ask from you is to have patience with the issues I initiate trying to follow up on this PR.